app/src/main/java/com/example/yard/Activities/PostActivity.java

```diff
@@ -3,6 +3,7 @@
 import android.Manifest;
 import android.content.Intent;
 import android.content.pm.PackageManager;
+import android.database.Cursor;
```

Comment:
Delete this `Activities/PostActivity.java` before merging the PR. Only keep the one under folder `activities`